services/install-turtleblocksjs.sh

```diff
@@ -36,6 +36,7 @@ function supported_arms {
   echo "armv7l"
   echo "armv6l"
   echo "x86_64"
+  echo "arm64"
```

Comment:
`echo "arm64"` should be changed to `echo "aarch64"`

Reply:
thanks John!